tests/Unit/PDFObjectTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Smalot\PdfParser\Unit;
+
+use Smalot\PdfParser\Document;
+use Smalot\PdfParser\Page;
+use Smalot\PdfParser\PDFObject;
+use Tests\Smalot\PdfParser\TestCase;
+
+class PDFObjectTest extends TestCase
+{
+    public function testGetTextOnNullPage(): void
+    {
+        static::assertSame(' ', (new PDFObject(new Document()))->getText());
+    }
+
+    public function testGetTextOnPageWithoutContent(): void
+    {
+        $document = new Document();
+
+        static::assertSame(' ', (new PDFObject($document, null, null))->getText(new Page($document)));
```

Comment:
```suggestion
        $this->assertSame(' ', (new PDFObject(new Document()))->getText());
    }

    public function testGetTextOnPageWithoutContent(): void
    {
        $document = new Document();

        $this->assertSame(' ', (new PDFObject($document, null, null))->getText(new Page($document)));
```

Can you tell me why a white space is the default return value here?